### DISCUSSION TITLE
where the file server_config.yaml

### DISCUSSION THREAD

**Discussion by 502dxceit** [milvus-io/milvus#26507] [Q&A and General discussion] [answered]
excuse me, when I up the milvus container with docker-compose, I can't find the file server_config.yaml anywhere, especially I want to 
check the config index_building_threshold in milvus's config. After upping the container the Milvus file structure is not like which was showed at documents: https://milvus.io/docs/v1.0.0/milvus_config.md.
So, where can I fix the global config or how to find the config file?
Thanks for answering, appreciate!

**Comment by yhmo** [comment]
https://milvus.io/docs/v1.0.0/milvus_config.md --------------- This url is for Milvus v1.x, not 2.x

If you are using Milvus v2.x:
- use `docker ps` to show the containers, find out the id of the Milvus standalone container
- use `docker exec -it [container id] bash` to enter the Milvus standalone container
- you will see the config file in this path:   /milvus/configs/milvus.yaml

**Reply by yhmo** [reply, marked as answer]
If you want to change the configurations in this file, you have two choices:
1. Directly modify the `/milvus/configs/milvus.yaml` in the container, then restart the container
2. Or, follow the doc: https://milvus.io/docs/configure-docker.md
